react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { Redirect} from 'react-router-dom';
+import styles from '../../styles.module.scss';
+import Navbar from "../Navbar"
+import { connect } from "react-redux";
+import actionCreators from "../../../../../redux/actions";
+
+class Matches extends Component {
+  
+  async componentDidMount(){
+    const response = this.props.dispatch(actionCreators.getMatches());
+    //this.props.isLoading = true;
+    //const response = await TicTacToeService.getMatches();
+    if(response.ok){
+      this.props.matches = response.data;
+      this.props.isLoading = false;
+    }
+  }
+
+  render() {
+    if(!window.localStorage.getItem("token")){
+      return <Redirect to="/"/>;
+    }
+
+    const matchesHistory = this.props.matches;
+    const matches = matchesHistory.map(game => {
+      const match = "Player 1: " + game.player_one + " || Player 2: " +  game.player_two + " || Winner: " + game.winner;
+      return (
+        <li key={game.id} >
+          {match}
+        </li>
+      );
+    });
+    var Spinner = require('react-spinkit');
+
+    if(this.props.isLoading){
+      return (<Spinner name='folding-cube' />);
+    }else{
```

Comment:
Unnecessary else since you are returning above:

```js
if (this.props.isLoading) {
  return <Spinner name='folding-cube' />;
}

return (
   ... 
);
```
You can also remove the parenthesis around the spinner.